src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -664,6 +664,9 @@ def load_arguments(self, _):
             c.argument('yes', options_list=[
                        '--yes', '-y'], help='Do not prompt for confirmation.', action='store_true')
 
+    with self.argument_context('aks trustedaccess role') as c:
+        c.argument('list', required=True, action='store_true', options_list=['--list', '-l'],
```

Comment:
may not need the action='store_true'?
correct me if i'm wrong @FumingZhang

Reply:
Yes, `list` should be the sub-command not an option/parameter.

From your entry function `aks_trustedaccess_roles_list`, location should be the option/parameter.

Reply:
Yes!